docs/api-testing/use-cases/use-drive.md

```diff
@@ -0,0 +1,363 @@
+---
+id: use-drive
+title: Using a File in your Tests
+sidebar_label: Using File from Vault Drive
+description: 'How to use the files from the Drive in your tests'
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+You can use files stored in the Vault drive in your test in several ways. This guide explains how you can use files stored in the Vault drive as a data source in your tests.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/).
+
+## Using a File as Datasource
+
+A very common scenario where you need an external file to be used in your test is when you have to generate a lot of different inputs inside your tests. In this case, it is difficult to add all your data inside the Input Sets because it would take a lot of time and often you might need to replace your data with a new set.
```

Comment:
```suggestion
A common scenario for using an external file in your test is when you have to generate a large number of different inputs inside your tests. In this case, it is difficult to add all your data inside the Input Sets because it would take a lot of time and often you might need to replace your data with a new set.
```